urllib3/gae_connectionpool.py

```diff
@@ -0,0 +1,96 @@
+from .connectionpool import (
+    ConnectionPool,
+    _Default,
+    get_host)
+
+from .exceptions import (
+    ProtocolError,
+    HostChangedError,
+    LocationValueError,
+    MaxRetryError,
+    SSLError,
+    TimeoutError,
+    InsecureRequestWarning,
+)
+
+from .packages.six import BytesIO
+from .request import RequestMethods
+from .response import HTTPResponse
+from .util.timeout import Timeout
+from .util.url import Url
```

Comment:
They are copy/pasted from connectionpool.py. I'll do a sweep and remove all unused imports once I make sure everything is covered functionally.